templates/csi.yaml.j2

```diff
@@ -371,6 +382,15 @@ spec:
               mountPath: "/var/lib/gluster"
             - name: gluster-dev
               mountPath: "/dev"
+            - name: varlog
+              mountPath: "/var/log/gluster"
+        - name: kadalu-logging
+          image: busybox
+          command: ["/bin/sh"]
+          args: ["-c", "while true; do logcnt=$(/bin/ls /var/log/gluster/ | wc -l); if [ \${logcnt} -gt 0 ]; then break; fi; sleep 5; done; tail -f /var/log/gluster/*.log"]
```

Comment:
@aravindavk can you think of a better way to do the wait and run? Should we have our own image (busybox + script) instead?

Reply:
mainly because this is erroring out with below msg:

```
Traceback (most recent call last):
  File "/kadalu/main.py", line 465, in <module>
    main()
  File "/kadalu/main.py", line 448, in main
    deploy_csi_pods(core_v1_client)
  File "/kadalu/main.py", line 394, in deploy_csi_pods
    execute(KUBECTL_CMD, "create", "-f", filename)
  File "/kadalu/kadalulib.py", line 60, in execute
    raise CommandException(proc.returncode, out.strip(), err.strip())
kadalulib.CommandException: [1] b'clusterrole.rbac.authorization.k8s.io/external-attacher-runner created\nclusterrolebinding.rbac.authorization.k8s.io/csi-attacher-role created\nstatefulset.apps/csi-attacher created\nclusterrole.rbac.authorization.k8s.io/csi-nodeplugin created\nclusterrolebinding.rbac.authorization.k8s.io/csi-nodeplugin created' b'error: error parsing /kadalu/templates/csi.yaml: error converting YAML to JSON: yaml: line 83: found unknown escape character'
```

Reply:
This looks good to me.

Reply:
Found this. `-F` option to tail instead of `-f` https://unix.stackexchange.com/questions/31237/how-to-start-tailing-a-file-that-has-not-been-yet-created

Reply:
Removing `\` before ${logcnt} is showing as valid yaml. Checked online here http://www.yamllint.com/

Reply:
I noticed `-F` can't be used in our case.. as we are giving *.log. (-F uses 'name' option, and hence didn't open the file if it was created after the command started).

Noticed that I have to escape twice as its YAML. If I remove the escape, the logcnt was treated as an variable while constructing the string itself. Let me check if \\ solves. Next Iteration, will remove \\ (if this fails).